webpack/build.js

```diff
@@ -25,6 +27,34 @@ module.exports = function build(stripesConfig, options) {
     if (options.sourcemap) {
       config.devtool = 'source-map';
     }
+    if (options.createDll && options.dllName) { // Adjust build to create Webpack DLL
+      config.entry = {};
+      config.entry[options.dllName] = options.createDll.split(',');
+      config.output.library = '[name]';
+      config.output.filename = '[name].[hash].js';
+      config.plugins.push(new webpack.DllPlugin({
+        name: '[name]',
+        path: path.join(options.outputPath, '[name]-manifest.json'),
+      }));
+    }
+    if (options.useDll) { // Consume Webpack DLL
+      // Recommended by https://engineering.invisionapp.com/post/optimizing-webpack/ for performance
+      config.cache = true;
+      config.devtool = 'eval';
+      config.dependencies = options.useDll.split(',');
+
+      for (let dependency in options.dependencies) {
+        config.plugins.push(new webpack.DllReferencePlugin({
+          context: options.outputPath,
+          manifest: require(path.join(options.outputPath, `${dependency}-manifest.json`))
```

Comment:
So, I should probably add a new parameter `--dllPath` for this with default to be `outputPath` since that is very convenient for someone creating a DLL and then consuming it in a subsequent command.

Reply:
Adjusted `--useDll` to support paths.